libqtile/log_utils.py

```diff
@@ -86,7 +86,7 @@ def init_log(log_level=WARNING, log_path='~/.%s.log', log_truncate=False,
     # If we have a log path, we'll also setup a log file
     if log_path:
         try:
-            log_path = log_path % 'qtile'
+            log_path %= 'qtile'
```

Comment:
Ooh, that's interesting, I didn't know string formatting could be a compound assignment operator